commit 0ac6bf0cdb076baca8b458affdfc25721dff2e86
Author: Boris Brezillon <boris.brezillon@free-electrons.com>
Date:   Fri Dec 2 14:48:07 2016 +0100

    drm/vc4: Fix ->clock_select setting for the VEC encoder
    
    PV_CONTROL_CLK_SELECT_VEC is actually 2 and not 0. Fix the definition and
    rework the vc4_set_crtc_possible_masks() to cover the full range of the
    PV_CONTROL_CLK_SELECT field.
    
    Signed-off-by: Boris Brezillon <boris.brezillon@free-electrons.com>
    Signed-off-by: Eric Anholt <eric@anholt.net>
    (cherry picked from commit ab8df60e3a3b68420d0d4477c5f07c00fbfb078b)

Index: linuxtest/dummy/rpi_1166_0ac6bf0cdb076baca8b458affdfc25721dff2e86.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1166_0ac6bf0cdb076baca8b458affdfc25721dff2e86.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
